--- open-invaders-0.2/Makefile.am~	2007-06-11 14:20:53.000000000 +0000
+++ open-invaders-0.2/Makefile.am	2007-06-25 23:23:17.000000000 +0000
@@ -3,17 +3,17 @@
 ## If you don't want it to overwrite it,
 ## 	Please disable it in the Anjuta project configuration
 
-SUBDIRS = include src data
+SUBDIRS = src data
 
 open_invadersdocdir = ${prefix}/doc/open-invaders
-open_invadersdoc_DATA = \
-	README\
-	COPYING\
-	AUTHORS\
-	ChangeLog\
-	INSTALL\
-	NEWS\
-	TODO
+#open_invadersdoc_DATA = \
+#	README\
+#	COPYING\
+#	AUTHORS\
+#	ChangeLog\
+#	INSTALL\
+#	NEWS\
+#	TODO
 
 EXTRA_DIST = $(open_invadersdoc_DATA)
 
