--- open-invaders-0.3/Makefile.am~	2007-10-29 12:54:15.000000000 +0100
+++ open-invaders-0.3/Makefile.am	2007-11-11 16:17:14.000000000 +0100
@@ -6,13 +6,13 @@
 	data
 
 open_invadersdocdir = ${prefix}/doc/open-invaders
-open_invadersdoc_DATA = \
-	README\
-	COPYING\
-	AUTHORS\
-	ChangeLog\
-	INSTALL\
-	NEWS
+#open_invadersdoc_DATA = \
+#	README\
+#	COPYING\
+#	AUTHORS\
+#	ChangeLog\
+#	INSTALL\
+#	NEWS
 
 EXTRA_DIST = $(open_invadersdoc_DATA)
 
